nixos/modules/services/networking/wireguard.nix

```diff
@@ -227,13 +227,16 @@ let
 
             (optionals (values.allowedIPsAsRoutes != false) (map (peer:
             (map (allowedIP:
-            "${ipCommand} route replace ${allowedIP} dev ${name} table ${values.table}"
+            if (allowedIP == "0.0.0.0/0") || (builtins.match "[0:]+/0" allowedIP != null)
+            then "${pkgs.wireguard}/bin/wg-default add ${name} ${allowedIP}"
```

Comment:
Please do *not* ship distro-specific tools that use the `wg-` prefix namespace. This will likely clash with upstream.

Instead, call your utility something like: `nixos-set-wg-default-route` -- long, descriptive, distro-specific.

Reply:
Also, this added behavior that isn't configurable and magically kicks in when specific allowedIPs are set is undesirable because not everyone would want to use fwmarks and ip rules in this case.